src/Symfony/Bridge/Twig/NodeVisitor/TranslationNodeVisitor.php

```diff
@@ -92,6 +93,13 @@ protected function doEnterNode(Node $node, Environment $env)
                 $node->getNode('body')->getAttribute('data'),
                 $node->hasNode('domain') ? $this->getReadDomainFromNode($node->getNode('domain')) : null,
             ];
+        } elseif ($node instanceof FilterExpression &&
+            'trans' === $node->getNode('filter')->getAttribute('value') &&
+            $node->getNode('node') instanceof ConcatBinary) {
```

Comment:
Can you format the test like done on line 79-84?

Reply:
What if one part of the concat is not a string but a variable?

Reply:
@fabpot If it's not a string, we cannot extract it, but it's still better that it's now. Changed the code and added new tests.

Not sure if I should fix the failing Travis CI build